README.md

```diff
@@ -3,3 +3,5 @@
 [![Build Status](https://travis-ci.org/JuliaPlots/AbstractPlotting.jl.svg?branch=master)](https://travis-ci.org/JuliaPlots/AbstractPlotting.jl)
 
 An abstract plotting library part of the JuliaPlots/Makie.jl ecosystem.
+
+![logo](https://github.com/fungktt/AbstractPlotting.jl/blob/master/ap-logo.png)
```

Comment:
This file path should not include your GitHub name since this code will live in the master branch of this repo.

Reply:
I would also suggest moving the logo to the top of the README.md file.

Reply:
To clarify, you should essentially link to a location/url in the readme that does not exist right now i.e. "https://github.com/JuliaPlots/AbstractPlotting.jl/tree/master/assets/icons/ap-logo.png"

Reply:
but will exist once this code is merged into master.